app/_src/gateway/install/linux/amazon-linux.md

```diff
@@ -20,9 +20,51 @@ You can install {{site.base_gateway}} by downloading an installation package or
 > **Note:** {{site.base_gateway}} supports running on [AWS Graviton processors](https://aws.amazon.com/ec2/graviton/). It can run in all AWS Regions where AWS Graviton is supported.
 
 {% navtabs %}
-{% navtab Package %}
+{% navtab Package (AL2023) %}
+Install {{site.base_gateway}} on Amazon Linux 2023 from the command line.
 
-Install {{site.base_gateway}} on Amazon Linux from the command line.
+1. Download the Kong package:
+
+{% capture download_package %}
+{% navtabs_ee codeblock %}
+{% navtab Kong Gateway %}
+```bash
+curl -Lo kong-enterprise-edition-{{page.versions.ee}}.aws.amd64.rpm "{{ site.links.download }}/gateway-3.x-amazonlinux-2023/Packages/k/kong-enterprise-edition-{{page.versions.ee}}.aws.amd64.rpm"
```

Comment:
You'll need to update all of the download links in these examples to Cloudsmith links, similar to https://github.com/Kong/docs.konghq.com/pull/5841/files#diff-e21794e1a931347708305c4a77a4b38647adbfe75b1904f95455dfd508703ef3R144.